src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -144,8 +144,15 @@ public TestHostConnectionInfo GetTestHostConnectionInfo()
             var currentWorkingDirectory = Path.Combine(Path.GetDirectoryName(typeof(DefaultTestHostManager).GetTypeInfo().Assembly.Location), "..//");
             var argumentsString = " " + connectionInfo.ToCommandLineOptions();
 
-            // "TestHost" is the name of the folder which contain Full CLR built testhost package assemblies.
-            testHostProcessName = Path.Combine("TestHost", testHostProcessName);
+            // check in current location for testhost exe
+            var testhostProcessPath = Path.Combine(currentWorkingDirectory, testHostProcessName);
+
+            if (!File.Exists(testhostProcessPath))
```

Comment:
Similar check need to do before launching `QTAgent.exe` to support `dotnet test`.

Reply:
as discussed this is not needed. but I will confirm this once

Reply:
as dicussed with Arun offline we don't want to support Qtagent scenarios with dotnet test command line